Set LightCurveTemplateTemporalModel to not extrapolate by default (#6712)
gammapy/gammapy: docs/release-notes/6712.bug.rst

> Adapt `~gammapy.modeling.models.LightCurveTemplateTemporalModel` to return zero when evaluating outside the times in the template.

Reviewer: I would say something like "Allow configuration of interpolation option on LightCurve..."

Author: implemented.

> Make LightCurveTemplateTemporalModel return zero instead of linearly extrapolate when evaluating outside the times in the template.

Reviewer 2: ```suggestion
Adapt `~gammapy.modeling.models.LightCurveTemplateTemporalModel` to return zero when evaluating outside the times in the template.
```

> Allow configuration of interpolation options on  `~gammapy.modeling.models.LightCurveTemplateTemporalModel`, specifically updating defaults to return zero when evaluating outside the times in the template.

Reviewer 2: ```suggestion
Add configuration of interpolation options for  `~gammapy.modeling.models.LightCurveTemplateTemporalModel`, specifically to default to return zero when evaluating outside the times in the template.
```
